src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import java.io.*;
+import java.util.Scanner;
+
+public class Storage {
+    String path;
+    public Storage(String path) {
+        this.path = path;
+    }
+
+    public void initialize() throws IOException {
+        File newFile = new File(this.path);
+
+        newFile.getParentFile().mkdirs();
+
+        if (! newFile.exists()) {
+            newFile.createNewFile();
+        }
+        getTasks(newFile);
+    }
+
+    public void saveTasks() {
+        try {
+            FileWriter fileWriter = new FileWriter(path);
+            BufferedWriter bufferWriter = new BufferedWriter(fileWriter);
+            String tasks = "";
+            for (Task task : TaskList.getContent()) {
+                tasks += task.saveString() + "\n";
+            }
+            bufferWriter.write(tasks);
+            bufferWriter.close();
+
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+    }
+
+    public void getTasks(File file) {
```

Comment:
`get` might imply that the method returns something. Perhaps consider `loadTasks(File)`?